server/src/test/java/org/apache/druid/metadata/TestDerbyConnector.java

```diff
@@ -178,6 +178,17 @@ public PendingSegmentsTable pendingSegments()
     {
       return new PendingSegmentsTable(this);
     }
+
+    public void beforeBenchmark()
+    {
+      before();
+    }
+
+    public void afterBenchmark()
+    {
+      this.getConnector().deleteAllSegmentRecords();
+      after();
+    }
```

Comment:
We can avoid having these methods. Just make `before` and `after` public if needed.

Reply:
Removed.